src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -32,29 +31,14 @@ class DoctrineDataCollector extends DataCollector
     private array $connections;
     private array $managers;
 
-    /**
-     * @var array<string, DebugStack>
-     */
-    private array $loggers = [];
-
     public function __construct(
         private ManagerRegistry $registry,
-        private ?DebugDataHolder $debugDataHolder = null,
+        private DebugDataHolder $debugDataHolder,
```

Comment:
passing `null` needs to be deprecated before removing it

Reply:
Yes.

Reply:
Done in #50579